code/__DEFINES/antagonists.dm

```diff
@@ -71,3 +71,32 @@
 #define SPAWNTYPE_ROUNDSTART "roundstart"
 #define SPAWNTYPE_MIDROUND "midround"
 #define SPAWNTYPE_EITHER "either"
+
+// austation -- Adding Bloodsucker by Coolsurf6
```

Comment:
shameless, crediting yourself in code for a port

Reply:
A shame

Reply:
:winrar:

Reply:
I'm askign to be DMed when someone finds this shit code